galata/test/jupyterlab/low-vision.test.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+import { expect, galata, test } from '@jupyterlab/galata';
+
+test.use({
+  autoGoto: false,
+  mockState: galata.DEFAULT_DOCUMENTATION_STATE,
+  viewport: { height: 1024, width: 1280 }
+});
+
+test.describe('Low Vision Support Test', () => {
+  test('Visability of menu-bar at 400-zoom', async ({ page }) => {
+    await page.goto();
+    await page.evaluate('document.body.style.zoom=4.0');
```

Comment:
I did some digging into this and we should be aware that this method of zooming the page is not completely equivalent to using the browser zoom commands. Unfortunately, [Playwright does not support browser zoom and currently has no plans to do so](https://github.com/microsoft/playwright/issues/23357#issuecomment-1572737010), so this might be the best option we have.

Reply:
Yeah i did check if playwright had it's own function for this but there was none so thought it would be best to use this for now.